motor_control/Triple_HBridge/3PhaseMotorControl_IR2104.ino

```diff
@@ -0,0 +1,158 @@
+#include <ramping.h>
+
+int outHighPins[3] = { 10, 6, 4};
+int shutPins[3] = {9, 5, 3};
+int inPins[3] = {0,1,2};
+int throttlePin = A0;
+bool pin1_state = 0;
+bool pin2_state = 0;
+bool pin3_state = 0;
+int motor_speed = 0;
+int speed_max = 0;
+
+#define RAMP_SPEED 2;
+
+void setup() {
+ //For throttle controls
+// Serial.begin(9600);
+//  pinMode(A0, INPUT);
+ 
+  for(int j = 0; j < 3; j++) {
+    pinMode(shutPins[j], OUTPUT);
+    pinMode(outHighPins[j], OUTPUT);
+//    pinMode(outLowPins[j], OUTPUT);
+    pinMode(inPins[j], INPUT);
+    analogWrite(shutPins[j], 0);
+  }
+      bool pin1_state = digitalRead(inPins[0]);
+      bool pin2_state = digitalRead(inPins[1]);
+      bool pin3_state = digitalRead(inPins[2]);
+
+      attachInterrupt(digitalPinToInterrupt(inPins[0]), State_Change_1, CHANGE); // initializes interrupt pin for every input
+      attachInterrupt(digitalPinToInterrupt(inPins[1]), State_Change_2, CHANGE); // initializes interrupt pin for every input
+      attachInterrupt(digitalPinToInterrupt(inPins[2]), State_Change_3, CHANGE); // initializes interrupt pin for every input
+     
+     
+    pinMode(throttlePin, INPUT);
+}
+
+
+void loop() {
+
+  if(analogRead(throttlePin) < 512){
+    for(int i = 0; i < 3; i++){
+      analogWrite(outHighPins[i], 0);
+      digitalWrite(shutPins[i], HIGH);

```

Comment:
Let's try to minimize magic numbers:
512 - > `THROTTLE_NEUTRAL` or `THROTTLE_MED` (alternatively `kThrottleNeutral` or `kThrottleMed`)
etc